### ISSUE TITLE
🐛 [Bug Report] [WoTLK] InstallFullDB.sh error 

### ISSUE THREAD

**Issue by adellaci** [cmangos/issues#2769]
### Bug Details

~~~
> Trying to set last content update version in db
  - Apllying version changes ... FAILED!
>>> ERROR 1064 (42000) at line 1: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '-0074_arlokk bit(1) DEFAULT NULL' at line 1
>>> Failed to update content version!
~~~

### Steps to Reproduce

1. run InstallFullDB.sh
2.
3.
4.

### Expected behavior

Complete install without error

### Suggested Workaround

N/A

### Crash Log

N/A

### Core SHA1 Commit Hash

N/A

### Database SHA1 Commit Hash

35c978ca669ccae05cf4006ea026ccb77e512cb4

### Operating System

Ubuntu 20.04.3 Server LTS

### Client Version

3.3.5a (Wrath of the Lich King)

**Comment by cyberium** [comment]
Should be fixed thanks for pointing so fast ;)
Please confirm and  close

**Comment by adellaci** [comment]
Testing it now
One thing i did notice that is not working is in the first menu when you select "3. Try with current settings" nothing happen. you have to go through edit then "9. Return to main menu", Then select (I believe) "2. Install fresh WoTLK DB ....." 

It finished no error

**Comment by adellaci** [comment]
Realm & World started up without any new errors or warnings

**Comment by adellaci** [comment]
This weekend i'll manually backup my database, and go thru all options to test.

**Comment by adellaci** [comment]
@cyberium i do not know if this is related or not. Sunday morning EST i pulled, compiled, make install, and ran 'InstallFullDB.sh'. Then start realm and world. I ran about 6 to 8 quest in Wetlands, one of them to get crocodile hides. I was able to skin them.  

Core - Using commit hash(b3a3e78f3b505fca940d5711d5442d8b0fffb06c)
Database - Using commit hash(4c3034354647ad26c87091155392193532290923)

Started up, continued questing in Wetlands, along the way i wanted to grab some more leather, killed a crocodile. When i skinned in the progress bar showed, once it completed it displayed "Immune" and no drop of leather.
